app/src/forms/email/emailService.js

```diff
@@ -287,8 +287,9 @@ const service = {
   submissionReceived: async (formId, submissionId, body, referer) => {
     try {
       const { configData, contexts } = await buildEmailTemplate(formId, submissionId, EmailTypes.SUBMISSION_RECEIVED, referer, { body });
-
-      return service._sendEmailTemplate(configData, contexts);
+      if (contexts[0].to.length > 0) {
+        return service._sendEmailTemplate(configData, contexts);
```

Comment:
The specification of this function needs to return the result of the operation. By if-blocking this out, this function now returns nothing; if the caller function depends on the returned object, we're introducing a potential bug. We should have an else return { ...something}.